velox/exec/HashProbe.cpp

```diff
@@ -79,7 +79,11 @@ void extractColumns(
     }
     child->resize(rows.size());
     table->rows()->extractColumn(
```

Comment:
Wondering if it would be safer to add extractColumn API to the table itself?

```
table->extractColumn(rows.data(), rows.size(), projection.inputChannel, child)
```